tests/test_sequence.py

```diff
@@ -17,46 +17,47 @@ def test_table_with_sequence(engine_testaccount, db_parameters):
     )
     sequence_table.create(engine_testaccount)
     seq = Sequence(test_sequence_name)
-    conn = engine_testaccount.connect()
     try:
-        with conn.begin():
-            conn.execute(sequence_table.insert(), [{"data": "test_insert_1"}])
+        with engine_testaccount.connect() as conn:
+            with conn.begin():
+                conn.execute(sequence_table.insert(), [{"data": "test_insert_1"}])
 
-        select_stmt = select(sequence_table).order_by("id")
+            select_stmt = select(sequence_table).order_by("id")
 
-        with conn.begin():
-            result = conn.execute(select_stmt).fetchall()
-            assert result == [(1, "test_insert_1")]
-
-        autoload_sequence_table = Table(
-            test_table_name, MetaData(), autoload_with=engine_testaccount
-        )
+            with conn.begin():
+                result = conn.execute(select_stmt).fetchall()
+                assert result == [(1, "test_insert_1")]
```

Comment:
I think it'd also be a good idea to standardize result fetching to either always happen in, or out of the `begin` block. Just in case users take inspiration from our tests

Reply:
good suggestion, I updated all the tests to have assertion be within `begin()`